lib/matplotlib/transforms.py

```diff
@@ -2062,6 +2062,12 @@ def frozen(self):
         # docstring inherited
         return self
 
+    def __deepcopy__(self, memo):
+        # The identity transform does not need to lock out deepcopy
+        return self.frozen()
```

Comment:
Did you intentionally keep `return self.frozen()` instead of just `return self`?

Reply:
oops, no - just hadn't had my coffee yet